cwms-data-api/src/test/java/cwms/cda/api/DataApiTestIT.java

```diff
@@ -64,13 +65,15 @@
 import org.junit.jupiter.api.AfterAll;
 import org.junit.jupiter.api.AfterEach;
 import org.junit.jupiter.api.BeforeAll;
+import org.junit.jupiter.api.DisplayNameGeneration;
 import org.junit.jupiter.api.Tag;
 import org.junit.jupiter.api.extension.ExtendWith;
 import usace.cwms.db.jooq.codegen.packages.CWMS_ENV_PACKAGE;
 /**
  * Helper class to manage cycling tests multiple times against a database.
  * NOTE: Not thread safe, do not run parallel tests. That may be future work though.
  */
+@DisplayNameGeneration(IntegrationTestNameGenerator.class)
```

Comment:
I don't see this in the PR, was it defined previously?

Reply:
missed in initial submission